deploy/kubernetes/base/node_windows/node.yaml

```diff
@@ -22,13 +22,21 @@ spec:
       serviceAccountName: csi-gce-pd-node-sa-win
```

Comment:
I remember that this codebase has overlays https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/blob/master/deploy/kubernetes/README.md, this change should be only in 1.23+ (or whichever version had HPC enabled by default)